### ISSUE TITLE
Proposal: create an Istio Community Group

### ISSUE THREAD

**Issue by ams0** [istio/community#1272]
As other projects have dedicated community groups on community.cncf.io (list [here](https://community.cncf.io/chapters/)), I believe Istio can benefit tremendously in having a community home as well to host meetings, share them broadly within the CNCF community and beyond, and for more reasons (use of CNCF YouTube channels/playlist, hosting  in an easy-to-access calendar and so on). As a meetup organiser and CNCF Ambassador I'd be happy to spearhead this and help organise and setup a calendar of events (releases where we interview the release team, special events, end-users stories). The[ Istio Office Hours proposal](https://docs.google.com/document/d/13voR8qZwG8lKI2_xtvYhN6msBHp0MAdvlDXMUqQGFEM/edit) can also fall/be run on community.cncf.io.

cc @linsun @craigbox @kfaseela

**Comment by craigbox** [comment]
Thanks @ams0.  We'll discuss this at Steering tomorrow; I have tried to use community.cncf.io to link things that aren't hosted there, and it's not really designed for that.  I'd be open to using it to mirror/link to things but not use it as primary hosting.

**Comment by chaltenio** [comment]
Hi folks, 

I am very interested in participating and collaborating in everything related to the Istio community. Please keep me informed of what I can do to help, and I will certainly contribute some ideas as I get deeper into the project.

**Comment by cccsss01** [comment]
I'm interested, we definitely need this.

**Comment by linsun** [comment]
Thank you all for the interest, we have discussed this briefly in steering. @craigbox will take a lead on this effort and steering will try to work with community to arrange community meetings for each month.

**Comment by MiirzaBaig** [comment]
Hi folks,

Yes, it would be great if there were weekly community meetings, such that it would help a lot for newcomers and continuous participants.

I would also want to be part of this amazing community and contributing to it.

**Comment by kfaseela** [comment]
https://github.com/istio/community?tab=readme-ov-file#community-meeting

**Comment by kfaseela** [comment]
@linsun : @srampal has created this "Istio Office Hours" proposal, can we formalize this, and make this part of community meetings? People who sign up here, can give the sessions as part of the community meetings every month? https://docs.google.com/document/d/1fRbMPIWidrY_Y1pWO31pQBbtqYc6rIJGn2pijBSV1dE/edit

**Comment by linsun** [comment]
Hi @kfaseela given we don't have topic for community meeting in Jan, I'm open to use community mtg for office hours.

If we do have enough topics for community meeting, I'd prefer split between user community meeting and developer/contributor community meeting (which is what @srampal is proposing for office hours to my understanding).

cc @istio/steering-committee to chime in

**Comment by linsun** [comment]
@srampal and @howardjohn thoughts on placing you in the next community meeting?  We can have it as early as next Thursday or the week after next thurs' 1pm.

**Comment by howardjohn** [comment]
The community meetings are for the community. Office hours are for developers. 

I don't know why we keep making this a big deal. Just stick a meeting on the calendar :-). If we cannot get agreement on how to schedule a meeting, I can just run black market office hours for folks who are interested.

**Comment by kfaseela** [comment]
@howardjohn : so should we have a separate meeting on the Istio calendar for office hours, and run this separately? Ofcourse we can do that, just that we need to formalize and then Sanjeev can drive.

**Comment by howardjohn** [comment]
I very much do not think we need to formalize this or have anyone drive... we are way over thinking this. If developers are in need of help, why are we putting so many artificial barriers to letting me (and others) get on a video call and help our developers? Just pick a time and lets do it :slightly_smiling_face:

**Comment by kfaseela** [comment]
> I very much do not think we need to formalize this or have anyone drive... we are way over thinking this. If developers are in need of help, why are we putting so many artificial barriers to letting me (and others) get on a video call and help our developers? Just pick a time and lets do it 🙂

it would be easier to advertise once we have this added to our calendar? Or we have random time slots based on the availability of speakers? That makes it difficult for people to track the event.

**Comment by cccsss01** [comment]
I'd like a set schedule/planned event, if there availability of whomever independent as there is always the community regardless.

**Comment by linsun** [comment]
I agree there is value of getting it scheduled and recorded.  Let us just get the office hour scheduled and stick with the schedule @srampal and @howardjohn 

@cccsss01 are you interested in contributing? If so, you should join Istio WG and ambient meeting, see: https://github.com/istio/community/blob/master/WORKING-GROUPS.md#working-group-meetings

**Comment by srampal** [comment]
> I agree there is value of getting it scheduled and recorded. Let us just
get the office hour scheduled and stick with the schedule @srampal
<https://github.com/srampal> and @howardjohn <https://github.com/howardjohn>

Agree Lin, thats where we were going anyway. We will start from next week.
